src/entities/inputValidator/composableStable/inputValidatorComposableStable.ts

```diff
@@ -34,7 +37,9 @@ export class InputValidatorComposableStable extends InputValidatorBase {
     }
 
     validateCreatePool(input: CreatePoolV2ComposableStableInput): void {
-        super.validateCreatePool(input);
+        validateCreatePoolTokens(input.tokens);
+        validateCreatePoolTokenConfig(input);
```

Comment:
`validateCreatePoolTokenConfig` seems to be relevant to v3 only and ComposableStable exists only on V2 so you should be able to remove this from here